Doc/library/sched.rst

```diff
@@ -20,8 +20,8 @@ scheduler:
    The :class:`scheduler` class defines a generic interface to scheduling events.
    It needs two functions to actually deal with the "outside world" --- *timefunc*
    should be callable without arguments, and return  a number (the "time", in any
-   units whatsoever). If time.monotonic is not available, the *timefunc* default
```

Comment:
Is it necessary to retain this sentence at all? It's already conveyed in the function prototype, and no special note is made for `delayfunc`.

I think maybe just remove the sentence entirely now that `timefunc`'s default value is no longer conditional.

Not sure if it's worth adding a `.. versionchanged:: 3.5` that says that `time.monotonic` is the default on all platforms now, since that's essentially covered in `time.monotonic` already, but I don't think it would *hurt*.

Reply:
@pganssle Yes, I think that's all agreeable.  Squashed the recent commit.